test/sources/compiler.json

```diff
@@ -0,0 +1,17 @@
+{
+  "contracts": "*",
+  "solcVersion": "0.6.0",
```

Comment:
Should this value and all contract pragma values track with solc in package.json? i.e. `0.6.1`

Reply:
Hi @marcgarreau   

It could - intended this config to be a convenience that people can adjust the values of when they produce a test case. The version specified here gets fetched from a remote - there's no conflict with the package.json solc - it's separate. 

The pragmas float on `pragma solidity ^0.6.0` - think this should be ok for any 6 series compiler.

Are you seeing an error?

Reply:
nope, just double-checking :shipit: